cluster.tf

```diff
@@ -15,9 +15,11 @@
  */
 
 locals {
-  cluster_name = "test-cluster-svpc"
-  bastion_name = format("%s-bastion", local.cluster_name)
-  bastion_zone = format("%s-a", var.region)
+  cluster_name             = "test-cluster-svpc"
+  bastion_name             = format("%s-bastion", local.cluster_name)
+  bastion_zone             = format("%s-a", var.region)
+  cluster_secondary_ranges = [for secondary in data.google_compute_subnetwork.subnetwork.secondary_ip_range : secondary.ip_cidr_range]
```

Comment:
Are we using this somewhere?

Reply:
Yeah its used in the local below, which is then used for the intra-cluster FW rule. Let me push an update, because I think we can make this cleaner using service accounts as source + destination for one rule, and then allowing traffic to masters with another

Reply:
scratch that, I forgot you can't use a service account as a destination for egress rules